usaspending_api/api_contracts/contracts/v1/references/hash.md

```diff
@@ -61,6 +66,10 @@ Restore Filters From URL Hash Data
                         "selectedLocations": {},
                         "selectedNAICS": {},
                         "selectedPSC": {},
+                        "naics_codes": {
+                            require: ["11"],
+                            exclude: []
```

Comment:
This won't be legal. Tinyshield will only allow an array of at least one element, or for the key to not be provided at all. In addition JSON requires key names to  be in quotes.
```suggestion
                            "require": ["11"]
```